proto/event/client/event.proto

```diff
@@ -94,6 +94,9 @@ enum ApiId {
   GET_EVALUATION = 1;
   GET_EVALUATIONS = 2;
   REGISTER_EVENTS = 3;
+  GET_FEATURE_FLAGS = 4;
+  GET_SEGMENT_USERS = 5;
+  SDK_GET_VARIATION = 100;
```

Comment:
I'm adding an ID to report the evaluation latency when running locally on the server SDK.
To keep things clean, we will use the `SDK` as a prefix for all new IDs referencing the SDK internal metrics.